integrations/phone/examples/README.md

```diff
@@ -0,0 +1,23 @@
+
+
+## Examples
+
+This folder contains a set of examples that are intended to demonstrate how to use various features of the Watson Assistant phone channel. Below is a list of these examples, along with a brief explanation of each and the different features that each example demonstrates.
+
+
+### [Common Interactions](./common-interactions/)
+
+This example contains Watson Assistant phone integration actions that demonstrate how to:
```

Comment:
Watson Assistant phone integration actions demonstrate: 

(rest of sentence can be cut because you already said above that these are examples, plus each bullet point says 'how to' so 'how to' here isnt needed)

Reply:
```suggestion
This example contains Watson Assistant phone integration actions demonstrate:
```